service/service.go

```diff
@@ -139,6 +141,10 @@ func New(options ...Option) (Owner, error) {
 		metrics.Freeze()
 	}
 
+	if svc.RuntimeMetrics() == nil {
```

Comment:
Good point. Why do we want this only for runtime metrics? We don't seem to have it for standard metrics. It is running in a subroutine, should not cause a latency impact.